website/docs/specification/DigitalIdentityAnchor.md

```diff
@@ -0,0 +1,242 @@
+| DIA-02 | DIA Issuer DID | The DIA issuer MUST use did:web (or a trusted variant such as did:tdw) as the DIA issuer and the web domain MUST match the well known domain of the issuing authority so that verifiers can confirm authority identity via public web records.| DIA issuer specification|
+| DIA-03 | Scheme registration | The DIA issuing authority SHOULD register the identity scheme (including the trusted issuer DIDs) with the UN/CEFACT identifier scheme registry so that verifiers can leverage UN maintained scheme metadata to simplify DIA discovery and verification.| See UNTP [Identity Resolver](IdentityResolver.md)|
+| DIA-04 | Subject DID List | The DIA MUST allow multiple subject DIDs to be linked to the same registered identity so long as proof of ownership is verified for each - so that members can rotate multiple active DIDs | `VerfiedDIDList` property|
+| DIA-05 | Scope List| The DIA MUST include a list of scope URIs that unambiguously define the authorised role(s) of the member in the register so that verifiers can confirm the scope of the membership. | `scopeList` property |
+| DIA-06 | Register Type| The DIA MUST specify the register type so that verifiers can understand the context of the `scopeList`| `registerType` property|
+| DIA-07 | DIA Discovery| The DIA SHOULD be discoverable given either the DID or the registeredID| [DIA Discovery](#dia-discovery)|
+
+The examples below help to clarify the application of DIA-05 and DIA-06.
+
+## Logical Model
+
+The Digital Identity Anchor logical model.
+
+![Digital Product Passport data model](DigitalIdentityAnchor.svg)
+
+### Core Vocabulary Documentation
+
+The [UNTP core types vocabulary](https://jargon.sh/user/unece/untp-core/v/0.5.0/artefacts/readme/render) defines the uniquely identified Linked Data entities such as Product, Location, Facility, Party, Standard, Regulation, Criteria, Declaration, Attestation, Endorsement. The Digital Identity Anchor only re-uses the UNTP core `VerifiableCredential` and `Identifier` classes.
+
+
+### DIA Documentation
+
+The [DIA documentation](https://jargon.sh/user/unece/DigitalIdentityAnchor/v/0.1.0/artefacts/readme/render) provides a technology-neutral definition of classes, properties and code lists in the DIA model.
+
+## Implementation Guidance
+
+This section provides sample JSON-LD snippets for each DIA component.
+
+### Digital Identity Anchor
+
+The Digital Identity Anchors is a Verifiable credential. Please refer to [DPP VC Guidance](DigitalProductPassport.md#verifiable-credential) for information about the use of the verifiable credentials data model for UNTP.  THe issuing party for the VC MUST be the authority that owns or operates the identity register. 
```

Comment:
```suggestion
The Digital Identity Anchors is a Verifiable credential. Please refer to [DPP VC Guidance](DigitalProductPassport.md#verifiable-credential) for information about the use of the verifiable credentials data model for UNTP.  The issuing party for the VC MUST be the authority that owns or operates the identity register. 
```

Reply:
This seems like a weak link. How do we guard against malicious issuers of this credential pointing at a legitimate public registry?

Reply:
I would rather see a pattern of including this DIA in a presentation. I would consider the [linked-vp spec](https://identity.foundation/linked-vp/#use-cases), leveraged as a [whois](https://identity.foundation/trustdidweb/#whois-linkedvp-service) mechanism